app/controlplane/pkg/data/workflowrun.go

```diff
@@ -181,9 +181,13 @@ func (r *WorkflowRunRepo) FindByIDInOrg(ctx context.Context, orgID, id uuid.UUID
 
 // SaveAttestation Saves the attestation for a workflow run in the database
 func (r *WorkflowRunRepo) SaveAttestation(ctx context.Context, id uuid.UUID, att *dsse.Envelope, digest string) error {
-	run, err := r.data.DB.WorkflowRun.UpdateOneID(id).
-		SetAttestationDigest(digest).
-		Save(ctx)
+	q := r.data.DB.WorkflowRun.UpdateOneID(id).
+		SetAttestationDigest(digest)
+	if att != nil {
+		// Set attestation when using old CLI versions
+		q.SetAttestation(att)
```

Comment:
but I thought that we wanted to store always bundle. Didn't we end up doing some sort of automatic wrapping on a bundle?

Reply:
We could. But even if we do it, old attestations must still be supported